framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -335,5 +335,21 @@ int cfg_get_all_prplmesh_wifi_interfaces(BPL_WLAN_IFACE *interfaces, int *num_of
     return RETURN_OK;
 }
 
+bool cfg_get_persistent_db_enable(bool &enable)
+{
+    int persistent_db_enable = DEFAULT_PERSISTENT_DB;
+
+    // persistent db value is optional
+    if (cfg_get_param_int("persistent_db=", persistent_db_enable) < 0) {
+        MAPF_DBG("cfg_get_persistent_db_enable: Failed to read persistent-db-enable parameter - "
+                 "setting default value\n");
```

Comment:
I don't see that on any of the prints on the file `\n` is added at the end. So probably it adds a new line by default.

Reply:
you are right - a copy-paste from the UCI implementation - removed it.